libs/header.cmake

```diff
@@ -1,12 +1,21 @@
 # Define the header-only Boost target
-add_library(Boost::boost INTERFACE IMPORTED GLOBAL)
+add_library(Boost_boost INTERFACE)
+add_library(Boost::boost ALIAS Boost_boost)
```

Comment:
To eliminate confusion, I think if you're going to adopt the `Boost_` convention you should probably convert *everything*, but maybe there's a functional reason for this. Just food for thought.

Reply:
The real targets are already named `Boost_*`. Rerefences between libraries are done using `Boost::`.
This change was made to allow the target with the header only libraries to be exported (since it was IMPORTED, nothing would happen).